src/main/java/core/basesyntax/Discount.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class Discount {
```

Comment:
lets remove this class from PR. You should test your code, but you should not push test classes to PR's